src/main/java/seedu/cafectrl/ui/Ui.java

```diff
@@ -82,16 +82,20 @@ public void showDeleteMessage(Dish selectedDish) {
         showToUser("Okay! " + selectedDish.getName() + " is deleted! :)");
     }
 
+    public void showBuyIngredientHeader() {
+        showToUser("Added to stock:");
+    }
+
     /**
      * Shows ingredient to buy message to user
      *
      * @param ingredient Ingredient to be bought
      */
     public void showBuyIngredientMessage(Ingredient ingredient) {
-        String header = "Added to stock: \n";
         String ingredientString = "Ingredient: " + ingredient.getName()
-                                    + "\tQty: " + ingredient.getQty();
-        showToUser(header + ingredientString);
+                                    + "\t\tQty: " + ingredient.getQty()
+                                    + ingredient.getUnit();
```

Comment:
perhaps change the indentation of these lines to have 8 white spaces from parent line